### PR TITLE
FB util file renamed and fixed `originalTimestamp` override

### PR DESCRIPTION
**Pull request by MoumitaM**
## Description of the change

- Fixed util file name typo
- Prevented original timestamp being overridden by current timestamp

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/488)

### CODE REVIEW

File: utils/EventRepository.js [hunk 1]
```diff
@@ -63,7 +63,9 @@ class EventRepository {
    */
   enqueue(rudderElement, type) {
     const message = rudderElement.getElementContent();
-    message.originalTimestamp = getCurrentTimeFormatted();
+    message.originalTimestamp = message.originalTimestamp
+      ? message.originalTimestamp
+      : getCurrentTimeFormatted();
```
Comment (saikumarrs) [inline utils/EventRepository.js:68]:
```suggestion
    message.originalTimestamp = message.originalTimestamp || getCurrentTimeFormatted();
```

File: integrations/FacebookPixel/utils.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function getEventId(message) {
```
Comment (saikumarrs) [inline integrations/FacebookPixel/utils.js:1]: Raise a separate PR for these changes.

**Review by saikumarrs — CHANGES_REQUESTED** [review]
PR needs to be split into two as the changes included are unrelated to each other.

**Comment by MoumitaM** [comment]
If I go back to the previous state for the util file name, It won’t allow me to push the changes. For this time can we go with one PR for both the fix? @saikumarrs

File: integrations/FacebookPixel/utils.js (hunk 2 above)
Comment (saikumarrs) [reply integrations/FacebookPixel/utils.js:1]: > If I go back to the previous state for the util file name, It won’t allow me to push the changes. For this time can we go with one PR for both the fix? @saikumarrs

Update the PR title accordingly.

Comment (saikumarrs) [reply integrations/FacebookPixel/utils.js:1]: Resolved it myself.

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**